modules/mobile-agents/android/client/plugins/IDPProxy/src/org/wso2/mdm/agent/proxy/utils/ServerUtilities.java

```diff
@@ -196,15 +242,48 @@ public static HttpRequestBase buildHeaders(HttpRequestBase httpRequestBase,
 		} catch (IOException e) {
 			responseParams.put(Constants.SERVER_RESPONSE_BODY, "Internal Server Error");
 			responseParams.put(Constants.SERVER_RESPONSE_STATUS,
-			                   Constants.INTERNAL_SERVER_ERROR);
+					Constants.INTERNAL_SERVER_ERROR);
 			throw new IDPTokenManagerException("Server connectivity failure.", e);
 		}
-		
+
+		return responseParams;
+	}
+
+	public static Map<String, String> sendPutRequest(String url, String params, Map<String,
+			String> headers) throws IDPTokenManagerException {
+
+		HttpPut httpPut = new HttpPut(url);
+		HttpClient httpClient = getCertifiedHttpClient();
+		HttpPut httpPutWithHeaders = (HttpPut) buildHeaders(httpPut, headers);
```

Comment:
What you've implemented in this particular line is something like decorating an instance of HttpPut class. Instead of creating a new variable called httpPutWithHeaders, you could have simply assigned the decorated instance to the same variable "httpPut" as the original assignment is never used within the downstream code.